### PR TITLE
Update React peerDependency to allow working with v15.x (15.0.1)

### PR DESCRIPTION
**Pull request by antoinerousseau**
Current issue:

    $ npm i --save react@15
    npm ERR! peerinvalid The package react@15.0.1 does not satisfy its siblings' 
    peerDependencies requirements!
    npm ERR! peerinvalid Peer react-big-calendar@0.9.8 wants react@^0.14.0-beta1

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -36,8 +36,8 @@
     "release": "release"
   },
   "peerDependencies": {
-    "react": "^0.14.0-beta1",
-    "react-dom": "^0.14.0-beta3"
+    "react": "^0.14.0 || ^15.0.1",
```
Comment (jquense) [inline package.json]: lets keep this at `15.0.0`, no need to exclude anyone here

**Comment by jquense** [comment]
small comment, and good to go, thanks!

File: package.json (hunk 1 above)
Comment (antoinerousseau) [inline package.json]: you're right

**Comment by antoinerousseau** [comment]
\o/

**Pull request merged**